### DISCUSSION TITLE
first Installation stops by "Enabling Imagick..." in nextcloud-aio-nextcloud container

### DISCUSSION THREAD

**Discussion by L0rz** [nextcloud/all-in-one#4512] [Questions] [answered]
### Steps to reproduce
1. i hostet a Cloud-server by Hetzner config( CX31 whit Debian 12)
2. i install docker whith : curl -fsSL https://get.docker.com | sudo sh
3. i deploy a Portainer-agend to configurate the docker-container
4. and install nextcloud AIO whit https://github.com/nextcloud/all-in-one/blob/main/compose.yaml
5. i Connectet per web to mastercontainer and start the installation
6. the "nextcloud-aio-nextcloud" shows only this log and stops:
```
 Waiting for database to start...
Connection to nextcloud-aio-database (172.19.0.5) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
Enabling Imagick...
```
I try it whit a absolut many new installations, and i can replikate it every time 

### Host OS 
shared vCPU whit 2 Cores and 8 GB RAM

#### Nextcloud AIO version 
latest
#### Current channel 
testet wit lates and beta

#### Other valuable info 
mastercontainer:
```
⚠️ Important: do always use an ip-address if you access this port and not a domain as HSTS might block access to it later!
If your server has port 80 and 8443 open and you point a domain to your server, you can get a valid certificate automatically by opening the Nextcloud AIO Interface via:
https://your-domain-that-points-to-this-server.tld:8443
INF ts=1712410214.5481315 msg=using provided configuration config_file=/Caddyfile config_adapter=
[Sat Apr 06 13:30:14.567170 2024] [mpm_event:notice] [pid 103:tid 139674069072648] AH00489: Apache/2.4.58 (Unix) OpenSSL/3.1.4 configured -- resuming normal operations
[Sat Apr 06 13:30:14.571050 2024] [core:notice] [pid 103:tid 139674069072648] AH00094: Command line: 'httpd -D FOREGROUND'
[06-Apr-2024 13:30:14] NOTICE: fpm is running, pid 110
[06-Apr-2024 13:30:14] NOTICE: ready to handle connections
```
nextcloud
```
Waiting for database to start...
Connection to nextcloud-aio-database (172.19.0.5) 5432 port [tcp/postgresql] succeeded!
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
Enabling Imagick...
```

**Comment by szaimen** [comment, marked as answer]
Hi, apparently the container is not able to download something from the alpine pacakge repository. This is something you need to fix in your network infrastructure.

**Reply by L0rz** [reply]
Okay, thx for the hint.
I Start a Installation whit a Public adress and no OPNsense as Firewall and it works.
I test a Instalation whit Firewall disabled ( pfctl -d) and it works... 
But no bocket packets in Log files.

Case Closed.

**Reply by HenryChan-Git** [reply]
The firewall has been turned off, but I am still stuck in this step. Is there any way to manually download what I need

**Reply by HenryChan-Git** [reply]
docker-php-ext-enable imagick >/dev/null;
I suggest changing all outputs to temporary files and automatically deleting them after successful installation, so that you can check the error log information when the installation is stuck

**Reply by L0rz** [reply]
In my case it helps to set the MTU of my Docker network to the same es the Servers network

com.docker.network.driver.mtu  1450

**Reply by DIS-DaIsserSchon** [reply]
Hey everybody. I have the same problem. @L0rz How I can set the MTU of the docker network individuell? Thank you for answer!

**Reply by L0rz** [reply]
I change to this at the end of my docker Compose file :

```
networks:
   nextcloud-aio:
      driver: bridge
      driver_opts:
        com.docker.network.driver.mtu: 1450
        name: nextcloud-aio # This line is not allowed to be changed as otherwise the created network will not be used by the other containers of AIO
```
